tests/osgi-tests/src/test/resources/carbon-context/deployment.yaml

```diff
@@ -0,0 +1,53 @@
+################################################################################
+#   Copyright (c) 2016, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+#
+#   Licensed under the Apache License, Version 2.0 (the "License");
+#   you may not use this file except in compliance with the License.
+#   You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+#   Unless required by applicable law or agreed to in writing, software
+#   distributed under the License is distributed on an "AS IS" BASIS,
+#   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#   See the License for the specific language governing permissions and
+#   limitations under the License.
+################################################################################
+
+  # Carbon Configuration Parameters
+wso2.carbon:
+    # value to uniquely identify a server
+    # THIS IS A MANDATORY FIELD
+  id: carbon-kernel
+    # server name
+  name: WSO2 Carbon Kernel
+    # server version
+  version: 5.2.0-SNAPSHOT
```

Comment:
These values need to be removed and only the minimum configuration values should be there in deployment.yaml